.rspec

```diff
@@ -1,2 +1,3 @@
 --color
 --order random
+--format documentation
```

Comment:
I feel strongly that the progress format is the way to go, was this changed for a reason?

Reply:
Cause I like it better cause I can see what specs are failing better with debug output and such. But removing it if you feel strongly.

Reply:
We should build jest-rspec - then you get all the Jest tooling for Ruby

Reply:
Yeah, during development I sometimes like documentation too, but this setting goes all over, including on CI, where I really dislike it. I have a shell thingy for this too:

https://github.com/jonallured/dotfiles/blob/master/files/.zshrc.local#L7